packages/commands/README.md

```diff
@@ -15,133 +15,123 @@ This tool needs to be built at least once before running.
 
 If you plan on developing commands, you can run `npm run watch:dist` to start a watcher, which will rebuild any time the files in `src` change.
 
-## Examples
+## Commands
 
-**create-user**
-
-```
-audius-cmd create-user freddie_mercury
-```
-
-**edit-user**
-
-```
-audius-cmd edit-user freddie_mercury --bio "easy come easy go"
-```
-
-**upload-track**
-
-```
-audius-cmd upload-track --from freddie_mercury --title "Bohemian Rhapsody"
 ```
+Usage: audius-cmd [options] [command]
 
-**edit-track**
-
-```
-audius-cmd edit-track <track-id> --title "Bohemian Rhapsody (Reprise)"
-```
-
-**create-playlist**
-
-```
-audius-cmd create-playlist <track-ids> --from freddie_mercury
-```
-
-**edit-playlist**
-
-```
-audius-cmd edit-playlist <playlist-id> --name "A Night at the Opera"
-```
+CLI interface for interacting with a local Audius network.
 
-**favorite-track**
+Options:
+  -h, --help                                     display help for command
 
-```
-audius-cmd favorite-track <track-id>
+Commands:
+  user                                           Commands that create or target a specific user
+  track                                          Commands that create or target a specific track
+  playlist                                       Commands that create or target a specific playlist
+  album                                          Commands that create or target a specific album
+  manager                                        Commands for managing account managers
+  auth-headers [options]                         Output auth headers (for use with curl: `curl -H @<(audius-cmd auth-headers)`)
+  claim-reward [options] <challengeId> <amount>  Claim a challenge reward
+  reward-specifier [options] <challengeId>       Get the specifier for a challenge
+  create-user-bank [options] [handle]            Create userbank for a user
+  mint [options] <amount>                        Mint $AUDIO or $USDC tokens
+  tip-reaction [options] <signature>             Send a tip reaction
+  withdraw-tokens [options] <account> <amount>   Send tokens from a user bank to an external address
+  help [command]                                 display help for command
 ```
 
-**unfavorite-track**
+### User subcommands
 
 ```
-audius-cmd unfavorite-track <track-id>
-```
+Usage: audius-cmd user [options] [command]
 
-**favorite-playlist**
-
-```
-audius-cmd favorite-playlist <playlist-id>
-```
+Commands that create or target a specific user
 
-**unfavorite-playlist**
+Options:
+  -h, --help                               display help for command
```

Comment:
Does this need to be updated to remove the `--` too?

Reply:
I'm just copy/pasting the output from the console, I don't think so

Reply:
looks like both are valid. the other change was probably unnecessary